shell/platform/common/cpp/text_input_model.h

```diff
@@ -110,7 +110,7 @@ class TextInputModel {
   int selection_extent() const { return selection_extent_; }
 
  private:
-  void DeleteSelected();
+  bool DeleteSelected();
```

Comment:
Can you add a declaration comment here, including what the return value means?

Reply:
Done!